### PR TITLE
Specify some steps in the release flow

### PR DESCRIPTION
**Pull request by enikonemeth**

### CODE REVIEW

File: docs/gitflow/gitflow.md [hunk 1]
```diff
@@ -1,87 +1,100 @@
...
+- Freeze _develop_ branch (do not merge any more pr to develop)
+- Create a release branch with [git flow](https://danielkummer.github.io/git-flow-cheatsheet/#release)
+- Increase the root package.json version to match with [semver](https://docs.npmjs.com/about-semantic-versioning)
+- Run `yarn lerna version --no-git-tag-version --include-merged-tags`
+  - bump packages version
+- Open pull request to _master_ branch
+- Do some testing (run unit and e2e tests)
+  - unit tests will run on github automatically -> you should only check the results
+  - e2e tests can be run on github and locally as well (on github start the AdminUI end-to-end tests workflow, locally you should run tests on https://e2e-service.test.sensenet.com repository)
+- Audit the netlify build with [Lighthouse](https://developers.google.com/web/tools/lighthouse#devtools), check for regressions, compare it with previous result
```
Comment (herflis) [inline docs/gitflow/gitflow.md]:
```suggestion
- Audit the netlify build with [Lighthouse](https://developers.google.com/web/tools/lighthouse#devtools), check for regressions, compare it with previous results
```

File: docs/gitflow/gitflow.md [hunk 2]
```diff
@@ -1,87 +1,100 @@
...
+  - lighthouse report should be run on the generated netlify url with https://dev.demo.sensenet.com repository. The recommended browser is chrome, it should be run in incognito window. In the settings you should also turn off the clear storage option. Please attach the result to the description of the release PR.
+- Check [relativeCI](https://app.relative-ci.com/projects/SpRCK0ViJsBVzSUtpFtk) for bundle size changes, compare it with the previous. Please attach the result to the description of the release PR.
```
Comment (herflis) [inline docs/gitflow/gitflow.md]:
```suggestion
- Check [relativeCI](https://app.relative-ci.com/projects/SpRCK0ViJsBVzSUtpFtk) for bundle size changes, compare it with the previous. Please attach results to the description of the release PR.
```

File: docs/gitflow/gitflow.md [hunk 3]
```diff
@@ -1,87 +1,100 @@
...
+- Write changelog and communicate
+- Lerna publish
+- Merge _hotfix_ into _master_
+- Merge _hotfix_ into _develop_
+- Remove _hotfix_ branch
+- Related issues --> Close
 
 # Environments
 
 Release
```
Comment (herflis) [inline docs/gitflow/gitflow.md]:
```suggestion
Test
```

File: docs/gitflow/gitflow.md [hunk 4]
```diff
@@ -1,87 +1,100 @@
...
-* Backend: https://netcore-service.test.sensenet.com/ - *master* branch - built daily
-* Frontend: https://admin.test.sensenet.com/ - *master* - built daily
+
+- Backend: https://netcore-service.test.sensenet.com/ - _master_ branch - built daily
+- Frontend: https://admin.test.sensenet.com/ - _master_ - built daily
 
 Staging
```
Comment (herflis) [inline docs/gitflow/gitflow.md]:
```suggestion
Release/Staging
```

**Review by herflis — CHANGES_REQUESTED** [review]

**Review by herflis — APPROVED** [review]

File: docs/gitflow/gitflow.md [hunk 5]
```diff
@@ -1,87 +1,108 @@
...
-* *master*: permanent branch for production environments
-* *develop*: permanent branch for aggregating feature branches
-* *feature*: temporary branch for features and bugfixes
-* *staging*: temporary branch for each release, used for creating near production QA environment with released backend
-* *hotfix*: temporary branch for urgent and high importance fixes
+
+- _master_: permanent branch for production environments
+- _develop_: permanent branch for aggregating feature branches
+- _feature_: temporary branch for features and bugfixes
+- _staging_: temporary branch for each release, used for creating near production QA environment with released backend
```
Comment (taki9) [inline docs/gitflow/gitflow.md]: - _staging_: temporary branch for each release, used for creating near production QA environment with released backend
```suggestion
- _release_: temporary branch for each release, used for creating near production QA environment with released backend
```

File: docs/gitflow/gitflow.md [hunk 6]
```diff
@@ -1,87 +1,108 @@
...
 
 # Environments
 
-Release
-* Backend: https://netcore-service.test.sensenet.com/ - *master* branch - built daily
-* Frontend: https://admin.test.sensenet.com/ - *master* - built daily
+Test
+
+- Backend: https://netcore-service.test.sensenet.com/ - _master_ branch - built daily
+- Frontend: https://admin.test.sensenet.com/ - _master_ - built daily
```
Comment (taki9) [inline docs/gitflow/gitflow.md]: develop branch, maybe on push?

File: docs/gitflow/gitflow.md [hunk 7]
```diff
@@ -1,87 +1,108 @@
...
+- Related issues --> Close
 
 # Environments
 
-Release
-* Backend: https://netcore-service.test.sensenet.com/ - *master* branch - built daily
-* Frontend: https://admin.test.sensenet.com/ - *master* - built daily
+Test
+
+- Backend: https://netcore-service.test.sensenet.com/ - _master_ branch - built daily
```
Comment (taki9) [inline docs/gitflow/gitflow.md]: develop I think

File: docs/gitflow/gitflow.md [hunk 8]
```diff
@@ -1,87 +1,108 @@
...
+- Merge _hotfix_ into _master_
+- Merge _hotfix_ into _develop_
+- Remove _hotfix_ branch
+- Related issues --> Close
 
 # Environments
 
-Release
-* Backend: https://netcore-service.test.sensenet.com/ - *master* branch - built daily
-* Frontend: https://admin.test.sensenet.com/ - *master* - built daily
```
Comment (taki9) [inline docs/gitflow/gitflow.md:79]: release environment is admin.sensenet.com
builds on master push?

**Review by taki9 — APPROVED** [review]

**Pull request merged**